### PR TITLE
Story/14382 Switch to BMG pandemic radar

### PR DESCRIPTION
**Pull request by naveeddotio**
## Description
Switching to BMG pandemic radar

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14382

### CODE REVIEW

**Review by KathrinLuecke — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift [hunk 1]
```diff
@@ -5,14 +5,23 @@
 import UIKit
 import OpenCombine
 
+enum pandemicRadarType {
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift]:
```suggestion
enum PandemicRadarProvider {
case rki
case bmg
}
```

Comment (flxschmidt) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift]: We don't use that type globally, so I guess a private type in an extension of `HomeLinkCardViewModel` would be nice.

**Review by flxschmidt — APPROVED** [review]
Nice, one suggestion, but not Approval preventing. :)

**Review by puneetmahali1 — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift (hunk 1 above)
Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCardView/HomeLinkCardViewModel.swift]: Good point, I have changed the scope to private and moved to private section.

**Pull request merged**